space_age.py

```diff
@@ -0,0 +1,29 @@
+class SpaceAge(object):
+    def __init__(self, seconds):
+        print('Earth = ' + '{0:.2f}'.format(seconds/60/60/24/365) + ' years old')
```

Comment:
@virtuallight i get rounded/integer here - why?

Reply:
In Python 2:
```
>>> 71997456/60/60/24/365
2
>>> '{0:.2f}'.format(71997456/60/60/24/365)
'2.00'
```

In Python 3:
```
>>> 71997456/60/60/24/365
2.283024353120244
>>> '{0:.2f}'.format(71997456/60/60/24/365)
'2.28'
```

The reason is that the division operator, in case of dividing two integers, in Python 2 returns the floor of the mathematical result of division e.g. `3 / 2` equals `1`, while in Python 3 in returns a "true division" e.g. `3 / 2` equals `1.5`.